content/tutorial-deep-learning-on-mnist.ipynb

```diff
@@ -0,0 +1,1017 @@
+{
```

Comment:
<p>Can you add a citation/reference for the fact that smaller numbers in the matrix result in a faster operation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Will try to find a quality source. Also, for intuition, I can include some basic math (e.g. a 28x28 matrix times the number of examples times the number of neurons per layer, etc etc 👻 = a high number of parameters to learn.

Reply:
I think the Wiki page is "looking for citation" :) But I know of a similar technique that expands on the idea further (batch norm). The research paper mentions standardization as a way to speed up training, because the incoming data that the network has to learn from has the same distribution as the one before it:

```diff
...
- - _Normalizing_ the image data.
+ - _Normalizing_ the image data: a [feature scaling]
+ (https://en.wikipedia.org/wiki/Feature_scaling#Application) procedure that can
+ speed up the neural network training process by standardizing the
+ [distribution of your input data](https://arxiv.org/pdf/1502.03167.pdf).
...
- > **Note:** Since the numbers inside matrices will be smaller, it should also make it faster for the neural network to learn from that data.
```

Reply:
<p>I think the references are good, but then I'd remove this note about smaller numbers causing faster learning, unless we have a good citation. I'm not a specialist in this so I don't necessarily see the connection :)</p>

Reply:
OK! I think the new version of this section looks better following the feedback below.